net-ftp/tftp-hpa/tftp-hpa-5.2-r2.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit systemd eutils toolchain-funcs
+
+DESCRIPTION="port of the OpenBSD TFTP server"
+HOMEPAGE="https://www.kernel.org/pub/software/network/tftp/"
+SRC_URI="https://www.kernel.org/pub/software/network/tftp/${PN}/${P}.tar.xz"
+
+LICENSE="BSD-4"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~ppc-macos"
+IUSE="ipv6 readline selinux tcpd +client +server"
+
+CDEPEND="
+	readline? ( sys-libs/readline:0= )
+	tcpd? ( sys-apps/tcp-wrappers )
+	!net-ftp/atftp"
+
+DEPEND="${CDEPEND}
+	app-arch/xz-utils"
+
+RDEPEND="${CDEPEND}
+	selinux? ( sec-policy/selinux-tftp )"
+
+PATCHES=(
+	"${FILESDIR}"/tftp-hpa-5.2-gcc-10.patch
+)
+
+src_prepare() {
+	eapply_user
+	sed -i "/^AR/s:ar:$(tc-getAR):" MCONFIG.in || die
+}
+
+src_configure() {
+	econf \
+		$(use_with ipv6) \
+		$(use_with tcpd tcpwrappers) \
+		$(use_with readline)
+}
+
+src_compile() {
+	emake version.h
+	emake -C lib
+	emake -C common
+	if use client; then
+		emake -C tftp
+	fi
+	if use server; then
+		emake -C tftpd
+	fi
+}
+
+src_install() {
+	dodoc README* CHANGES tftpd/sample.rules
+	emake INSTALLROOT="${D}" -C lib install
+	emake INSTALLROOT="${D}" -C common install
+
+	if use client; then
+		emake INSTALLROOT="${D}" SUB="lib common tftp" tftp install
+	fi
+	if use server; then
+
+		emake INSTALLROOT="${D}" SUB="lib common tftpd" tftpd install
+		# iputils installs this
+		rm "${ED}"/usr/share/man/man8/tftpd.8 || die
```

Comment:
This seems a bit odd. Does iputils not install a conflicting binary as well?

Reply:
Seems like a left over from a distant past. I don't why this was added (The comment exists in the original commit from Sat Aug 8 13:49:04 2015 -0700). I guess this comment be removed. 

```
equery f iputils
 * Searching for iputils ...
 * Contents of net-misc/iputils-20190709-r1:
/bin
/bin/arping
/bin/ping
/bin/ping4 -> ping
/bin/ping6 -> ping
/usr
/usr/bin
/usr/share
/usr/share/doc
/usr/share/doc/iputils-20190709-r1
/usr/share/doc/iputils-20190709-r1/README.md.bz2
/usr/share/locale
/usr/share/locale/ja
/usr/share/locale/ja/LC_MESSAGES
/usr/share/locale/ja/LC_MESSAGES/iputils.mo
/usr/share/man
/usr/share/man/man8
/usr/share/man/man8/arping.8.bz2
/usr/share/man/man8/ping.8.bz2
```